app/app_test.go

```diff
@@ -27,18 +27,21 @@ func newChildChain() *ChildChain {
 	return NewChildChain(logger, db, nil)
 }
 
-func InitTestChain(addr common.Address, cc *ChildChain) {
-	// Currently only initialize chain with one deposited UTXO
-	genState := GenesisUTXO{
-		Address:  addr.Hex(),
-		Denom:    "100",
-		Position: [4]string{"0", "0", "0", fmt.Sprintf("%d", 1)},
+// Creates a deposit of value 100 for each address in input
+func InitTestChain(addrs []common.Address, cc *ChildChain) {
```

Comment:
Prefer it to be `InitTestChain(cc *ChildChain, addrs ...common.Address)`

variable length args in go